cocos/network/HttpAsynConnection.m

```diff
@@ -73,7 +74,10 @@ - (void) connection:(NSURLConnection *)connection
     responseHeader = [[httpResponse allHeaderFields] copy];
 
     responseCode = httpResponse.statusCode;
-    if (responseCode != 200)
+    statusString = [[NSHTTPURLResponse localizedStringForStatusCode:responseCode] copy];
+    if(responseCode == 200)
+        statusString = @"OK";
+    if (responseCode/100 != 2)
```

Comment:
That is because when responseCode is 200 the statusString of iOS is different with other platform.